docs/theory.rst

```diff
@@ -16,79 +22,188 @@ The filter shape can be thought of in terms of the kernel of a convolution filte
 
 where :math:`f` is the function being filtered, :math:`G` is the filter kernel, and :math:`x'` is a dummy integration variable.
```

Comment:
Per Hussein's comments on our paper, I think we could add a parenthetical statement here saying that our filter with a Gaussian target does not produce _exactly_ the same results as convolution against a Gaussian kernel on the sphere.